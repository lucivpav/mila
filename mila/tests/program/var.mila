var a : integer;
var b : integer;
begin
  a := 1;
  b := 2;
  writeln(a);
  writeln(b);
end.
---input---
var a, b : integer;
var c, d : integer;
begin
  a := 1;
  b := 2;
  c := 3;
  d := 4;
  writeln(a);
  writeln(b);
  writeln(c);
  writeln(d);
end.
---input---
var a, b : integer;
    c, d : integer;
    e : integer;
begin
  a := 1;
  b := 2;
  c := 3;
  d := 4;
  e := 5;
  writeln(a);
  writeln(b);
  writeln(c);
  writeln(d);
  writeln(e);
end.
---input---
var a;
begin
  a := 1;
  writeln(a);
end.
---input---
var a : int;
begin
  a := 1;
  writeln(a);
end.
---input---
begin
  var a : integer;
  a := 1;
  writeln(a);
end.
---input---
begin
  writeln(a);
end.
