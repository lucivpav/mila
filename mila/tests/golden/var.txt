1
2
0
---output---
1
2
3
4
0
---output---
1
2
3
4
5
0
---output---
256
---output---
256
---output---
256
---output---
256
